geometric_features/aggregation/ocean/__init__.py

```diff
@@ -0,0 +1 @@
+from geometric_features.aggregation.ocean.subbasin import subbasin
```

Comment:
I just realized this won't work because the function and the module have the same name.  Hmm, maybe rename the module something like `subbasin_regions.py` and change this to:
```suggestion
from geometric_features.aggregation.ocean.subbasin_regions import subbasin
```